logging/monolog_regex_based_excludes.rst

```diff
@@ -16,7 +16,7 @@ configuration:
 
     .. code-block:: yaml
 
-        # app/config/config.yml
+        # config/packages/monolog.yaml
```

Comment:
I think `config/packages/prod/monolog.yaml`  is more proper - that's really where the user should modify things.

That might affect other logs docs too. In other words, I think we had it wrong before.